tests/jobs/misfit_preprocessor/unit/test_config.py

```diff
@@ -372,3 +389,54 @@ def test_default_observations(observation_keys):
 
     assert config.valid, config.errors
     assert sorted(observation_keys) == sorted(config.snapshot.observations)
+
+
+def test_new_config():
```

Comment:
Perhaps rename this, or check if the functionality is now covered by the refactored tests? It made a lot of sense when we started this work, but might be surpassed now?

Reply:
You are right, you even mentioned it in the chat. My bad when rebasing I probably missed it somehow.